### PR TITLE
JBPM-6621: Stunner - Add BPMN End Message Event

### PR DESCRIPTION
**Pull request by alepintus**
Hi @romartin  @hasys ,
End Message Event added.

Could you please take a look?

Thanks!

fyi @wmedvede

### CODE REVIEW

**Comment by romartin** [comment]
ok to test

**Comment by romartin** [comment]
Jenkins please retest this

**Comment by romartin** [comment]
hey @alepintus , now your turn.. :) you'll have to rebase this sorry, after merging @wmedvede 's PR this one complains. Thanks!

**Comment by alepintus** [comment]
Hi @romartin @hasys @wmedvede ,
PR rebased, could you please take a look?

Thanks!

**Comment by romartin** [comment]
@alepintus hey, please keep an eye on this PR, the build is failing due to other already known failures, not your fault, so just keep firing the regular and downstream builds, so once artifacts are generated @hasys will be able to review, and then I'll be able to merge :)
Thanks!

**Comment by romartin** [comment]
Jenkins please retest this

**Comment by romartin** [comment]
Jenkins execute full downstream build

**Comment by romartin** [comment]
Jenkins please retest this

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/java/org/kie/workbench/common/stunner/bpmn/definition/BaseEndEvent.java [hunk 1]
```diff
@@ -39,48 +39,49 @@
...
+        labels.add("to_task_event");
+        labels.add("from_task_event");
+        labels.add("fromtoall");
+        labels.add("choreography_sequence_end");
+        labels.add("Endevents_all");
+        labels.add("EndEventsMorph");
+        labels.add("cm_nop");
+    }
+
+    ;
```
Comment (hasys) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/java/org/kie/workbench/common/stunner/bpmn/definition/BaseEndEvent.java]: Useless semicolon.

**Comment by alepintus** [comment]
Hi @hasys ,
deleted semicolon.
 
Thanks!

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/java/org/kie/workbench/common/stunner/bpmn/definition/property/event/message/MessageEventExecutionSet.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+import org.kie.workbench.common.stunner.bpmn.forms.model.ComboBoxFieldType;
+import org.kie.workbench.common.stunner.core.definition.annotation.Name;
+import org.kie.workbench.common.stunner.core.definition.annotation.Property;
+import org.kie.workbench.common.stunner.core.definition.annotation.PropertySet;
+import org.kie.workbench.common.stunner.core.util.HashUtil;
+
+@Portable
+@Bindable
+@PropertySet
+@FormDefinition(startElement = "isInterrupting")
```
Comment (wmedvede) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/java/org/kie/workbench/common/stunner/bpmn/definition/property/event/message/MessageEventExecutionSet.java]: This execution set doesn't have the isInterrupting property, so the start element must be the messageRef property

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client/src/main/java/org/kie/workbench/common/stunner/bpmn/client/shape/def/EndEventShapeDef.java [hunk 3]
```diff
@@ -43,6 +44,7 @@
                          BPMNSVGViewFactory::endNoneEvent)
                     .put(EndSignalEvent.class,
                          BPMNSVGViewFactory::endSignalEvent)
+                    .put(EndMessageEvent.class, BPMNSVGViewFactory::endMessageEvent)
```
Comment (wmedvede) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client/src/main/java/org/kie/workbench/common/stunner/bpmn/client/shape/def/EndEventShapeDef.java]: If possible please try add the same indentation for this line as the other .put(xx lines in the same set.

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client/src/main/java/org/kie/workbench/common/stunner/bpmn/client/shape/def/EndEventShapeDef.java [hunk 4]
```diff
@@ -54,6 +56,7 @@
                     BPMNSVGGlyphFactory.END_NONE_EVENT_GLYPH);
                 put(EndSignalEvent.class,
                     BPMNSVGGlyphFactory.END_SIGNAL_EVENT_GLYPH);
+                put(EndMessageEvent.class, BPMNSVGGlyphFactory.END_MESSAGE_EVENT_GLYPH);
```
Comment (wmedvede) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client/src/main/java/org/kie/workbench/common/stunner/bpmn/client/shape/def/EndEventShapeDef.java]: same here

**Comment by alepintus** [comment]
Hi @wmedvede ,
code changed as suggested.

Thanks!

**Pull request merged**